src/AzureStorageQueueWorkerOptionsBuilder.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Likvido.Worker.AzureStorageQueue
+{
+    public class AzureStorageQueueWorkerOptionsBuilder
```

Comment:
What is the usefulness of this class?

Reply:
Mostly for future purposes, we can add more methods with extra more complicated validation instead of having all validation in the options. Keeping backward compatibility will be easier

Reply:
hmm... ok then